### PR TITLE
NEW SIQ_5 and SIQ_7, Spectrum Inquery Test Cases

### PR DESCRIPTION
**Pull request by assaneRed**
adding SIQ_5 and SIQ7

### CODE REVIEW

**Comment by assaneRed** [comment]
Hi Kate!
Can you check, please, and let me know if there are any others comments

Best regards

File: src/harness/testcases/WINNF_FT_S_SIQ_testcase.py [hunk 1]
```diff
@@ -130,4 +132,5 @@ def test_WINNF_FT_S_SIQ_7(self):
     response = self._sas.SpectrumInquiry(request, device_c_cert, device_c_key)['spectrumInquiryResponse'][0]
     # Check Spectrum Inquiry Response
     self.assertFalse('cbsdId' in response)
-    self.assertEqual(response['response']['responseCode'], 103)
\ No newline at end of file
+    self.assertTrue(response['response']['responseCode'] == 103 or
+                    response['response']['responseCode'] == 105)
```
Comment (kate-harrison) [inline src/harness/testcases/WINNF_FT_S_SIQ_testcase.py]: This should be 104, not 105.

**Comment by assaneRed** [comment]
Hi Kate! 
I changed the response code to "104"

Thanks

**Comment by kate-harrison** [comment]
Looks good! Please merge whenever you're ready.

**Pull request merged**